code/style.css

```diff
@@ -1,7 +1,215 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: lightpink;
+  color: black;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  background-image: url("header_small.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 10px;
+  padding: 18px;
+  border: 4px solid lightgray;
+  border-radius: 20%;
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  height: 500px;
+  width: 100%;
+}
+
+.primary-header {
+  text-shadow: 2px 2px lightpink;
+}
+
+.navigation-bar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  margin-top: 200px;
+}
+
+.navigation-bar a:link,
+a:visited {
+  color: black;
+  padding: 8px 12px;
+  text-decoration: none;
+  text-align: center;
+}
+
+.navigation-bar a:hover {
+  color: deeppink;
+}
+
+.placeholder {
+  margin: 10px;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 1300px;
+  width: 100%;
+}
+
+.form {
+  background: lightgray;
+  padding: 48px;
+  margin: 10px;
+  border: 4px solid gray;
+  border-radius: 20%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  text-align: left;
+  height: 700px;
+  width: auto;
+  object-fit: cover;
+  box-shadow: 20px 20px gray;
+}
+
+input {
+  margin: 8px;
+  padding: 5px;
+}
+
+.form-button {
+  width: 40%;
+  height: 30px;
+  margin: 8px;
+  padding: 5px;
+}
+
+.accordion {
+  background: lightgray;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: 10px;
+  margin-right: auto;
+  padding: 18px;
+  height: 550px;
+  width: auto;
+  border: 4px solid gray;
+  border-radius: 20%;
+  object-fit: cover;
+  box-shadow: 20px 20px gray;
+}
+
+.accordion-header {
+  color: black;
+  cursor: pointer;
+  margin: 10px;
+  padding: 18px;
+}
+
+.accordion-content {
+  display: none;
+  color: black;
+  cursor: pointer;
+  margin: 10px;
+  border-top: 2px solid lightgray;
+  padding: 18px;
+  text-align: left;
+  outline: none;
+  transition: 0.4s;
```

Comment:
Transitioning won't work with the display. [Further reference](https://stackoverflow.com/questions/3331353/transitions-on-the-css-display-property).

Reply:
Also consider adding a `max-width` to avoid "jumping"

Reply:
Removed it.